src/test/java/teammates/test/cases/logic/StudentsLogicTest.java

```diff
@@ -587,17 +587,16 @@ public void testEnrollLinesChecking() throws Exception {
     }
 
     /**
-     * This method returns the Exception message thrown when trying to build StudentList from invalid enrollment line.
-     * The method assumes that an Enrollment Exception is thrown, else the method fails with
-     * signalFailureToDetectException().
+     * Returns the Exception message thrown when trying to build StudentList from invalid enrollment line.
```

Comment:
Since this is describing what is returned, it can be placed in @return.
Would `@return EnrollException message thrown due to invalid enrollment line in html format` be better?